config.py

```diff
@@ -744,10 +744,18 @@
 """
 
 ###############################################################################
-# SOURCE SPACE PARAMETERS
-# -----------------------
+# SOURCE ESTIMATION PARAMETERS
+# ----------------------------
 #
 
+recreate_bem_surfaces: bool = False
```

Comment:
we have 2 ways to compute BEM surfaces: watershed and flash
that should be one parameter.

then we need another parameter that says if we force recompute.

wdyt?

Reply:
yes absolutely, will add

Reply:
I've included FLASH MRI support now